ui/src/authors/containers/AuthorPublicationsContainer.jsx

```diff
@@ -40,6 +43,7 @@ const stateToProps = state => ({
     'metadata',
     'facet_author_name',
   ]),
+  assignView: isCataloger(state.user.getIn(['data', 'roles'])),
```

Comment:
are we using only `cataloger` check in general?

Reply:
yes, for now only catalogers (and superusers) will see the UI.